unittest.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 26 13:53:41 2017
+
+@author: siddhant
+"""
+
+import unittest
+from point import Point
+import constants as c
+from line import Line
+
+
+p1 = Point(1.0, 2.0, 3.0)
+p2 = Point(2.0, 4.0, 6.0)
+l1 = Line(p1, p2)
+p3 = Point(1.0, 2.0, 3.0)
+p4 = Point(6.0, 12.0, 18.0)
+l2 = Line(p3, p4)
+
+class PointTestCase (unittest.TestCase):
+   
+   
```

Comment:
Please add tests which confirm no exception is thrown for the two valid ways to initialize a Point, by list and by individual arguments, and that does throw and exception if only one number is passed into the constructor.